src/main/resources/openapi/sqs/Notifications-sqs-messaging.yaml

```
@@ -847,6 +858,34 @@ components:
       required:
         - personalisation
         - channel
+    InviteToRegister:
+      title: InviteTORegister
+      description: The data an invite to register
+      type: object
+      properties:
+        freeText:
+          type: string
+    InviteToRegisterPersonalisation:
+      title: InviteToRegisterPersonalisation
+      type: object
+      description: Personalisation details for an invite to register
```

Since the above needs changing, think it's worth clarifying
```suggestion
      description: Personalisation details for an invite to register communication
```